scripts/update-current-status.py

```
@@ -402,37 +405,94 @@ def _update_roadmap() -> str:
-    if updated_status != original_status:
-        files_to_update.append(("docs/project/CURRENT_STATUS.md", CURRENT_STATUS, updated_status))
-
-    # Check ROADMAP.md
-    updated_roadmap = _update_roadmap()
-    original_roadmap = ROADMAP.read_text(encoding="utf-8")
-    if updated_roadmap != original_roadmap:
-        files_to_update.append(("docs/project/ROADMAP.md", ROADMAP, updated_roadmap))
+    files_to_update: list[tuple[str, pathlib.Path, str]] = []
+
+    if "lsp" in subsystems_to_run:
+        assert cov is not None and compliance_table is not None
```

Ruff (0.15.6)

[warning] Assertion should be broken down into multiple parts

Break down assertion into multiple parts

(PT018)